tests/trainer/test_dataloaders.py

```diff
@@ -412,8 +412,8 @@ def test_inf_val_dataloader(tmpdir, check_interval):
 
 
 @pytest.mark.parametrize('check_interval', [1.0])
-@pytest.mark.skip('TODO: speed up this test')
-def test_not_implemented_error_dataloader(tmpdir, check_interval):
+# @pytest.mark.skip('TODO: speed up this test')
+def test_not_implemented_error_val_dataloader(tmpdir, check_interval):
```

Comment:
same here, no test for "not implemented error"

Reply:
The name of the test refers CustomNotImplementedErrorDataloader which raises a NotImplementedError when __len__ is called. In this configuration training should take place. I am open for name suggestions.